openshift-template-dev.yml

```
@@ -136,3 +136,7 @@ parameters :
     description : Git repository with configuration
     value :
     required : true
+  - name : DOCKER_IMAGE
+    description : DockerHub repository with release-bot container dev code
+    value :
```

Can we provide a default value?

```suggestion
    value : usercont/release-bot:dev
```

---

Sure, make sense. 
I will also change a description text to
`DockerHub repository with release-bot development image`